lib/private/Security/CSP/ContentSecurityPolicy.php

```diff
@@ -64,6 +64,20 @@ public function setEvalScriptAllowed(bool $evalScriptAllowed) {
 		$this->evalScriptAllowed = $evalScriptAllowed;
 	}
 
+	/**
+	 * @return boolean
+	 */
+	public function isEvalWasmAllowed(): bool {
```

Comment:
its also null somewhere, so need to allow to return it?

```suggestion
	public function isEvalWasmAllowed(): ?bool {
```

Reply:
`@return boolean` should be `@return null|boolean`?

Reply:
You can remove the doc block. The function signature contains exactly the same information already